### PR TITLE
[LOOP-4847] align COB value

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-4847

### CODE REVIEW

File: Loop/View Controllers/CarbAbsorptionViewController.swift [hunk 1]
```diff
@@ -164,7 +164,7 @@ final class CarbAbsorptionViewController: LoopChartsTableViewController, Identif
                 let review = try await loopDataManager.fetchCarbAbsorptionReview(start: listStart, end: listEnd)
                 insulinCounteractionEffects = review.effectsVelocities.filterDateRange(chartStartDate, nil)
                 carbStatuses = review.carbStatuses
-                carbsOnBoard = carbStatuses?.getClampedCarbsOnBoard()
+                carbsOnBoard = loopDataManager.activeCarbs
                 carbEffects = review.carbEffects
```
Comment (nhamming) [inline Loop/View Controllers/CarbAbsorptionViewController.swift:168]: @ps2 this is where the COB value comes from for the carbs chart. Is it appropriate to re-use in the absorption view such that the value is aligned?

Comment (ps2) [reply Loop/View Controllers/CarbAbsorptionViewController.swift:168]: I think this is more a product question.  The `carbStatuses?.getClampedCarbsOnBoard()` should be more current. The `loopDataManager.activeCarbs` is less frequently updated (during a loop, generally).

File: Loop/View Controllers/StatusTableViewController.swift [hunk 2]
```diff
@@ -476,7 +476,8 @@ final class StatusTableViewController: LoopChartsTableViewController {
 
         // always check for cob
         carbsOnBoard = loopManager.activeCarbs?.quantity
-
+        print("!!! \(#function) carbsOnBoard \(String(describing: carbsOnBoard))")
```
Comment (ps2) [inline Loop/View Controllers/StatusTableViewController.swift]: Remove print?

**Review by ps2 — APPROVED** [review]
Code looks good to me.  Whether it's ok to have a little lag in active carbs to match the status screen is more a product decision.

**Pull request merged**